### PR TITLE
feat(trim): passing null to parameter

### PR DESCRIPTION
**Pull request by githubjeka**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️

### CODE REVIEW

File: framework/grid/Column.php [hunk 1]
```diff
@@ -149,7 +149,7 @@ protected function getHeaderCellLabel()
      */
     protected function renderFooterCellContent()
     {
-        return trim($this->footer) !== '' ? $this->footer : $this->grid->emptyCell;
+        return $this->header !== null && trim($this->footer) !== '' ? $this->footer : $this->grid->emptyCell;
```
Comment (bizley) [inline framework/grid/Column.php:152]:
```suggestion
        return $this->footer !== null && trim($this->footer) !== '' ? $this->footer : $this->grid->emptyCell;
```

**Comment by githubjeka** [comment]
```php
            [
                'format' => 'raw',
                'contentOptions' => ['class' => 'text-center'],
                'value' => function (Node $node) use ($filterModel) {
                    return Html::a(
                        '<i class="fas fa-2x fa-book"></i>',
                        [
                            '/journal/view',
                            'node_id' => $node->id,
                            'month' => $filterModel->month,
                            'year' => $filterModel->year,
                        ],
                        ['title' => 'Журнал', 'data-pjax' => 0]
                    );
                },
            ],
```

**Comment by WinterSilence** [comment]
@githubjeka no need append `null` to PHPDoc type, because we never pass `null` to this properties

**Comment by githubjeka** [comment]
> no need append `null` to PHPDoc type, because we never pass `null` to this properties

PHPDoc with `null` like syntax `public ?string $header = null;`

**Comment by WinterSilence** [comment]
`public ?string $header;` not same to `public string $header;`

**Pull request merged**